dependency_manager/README.md

```diff
@@ -88,6 +81,11 @@ If you are currently in the *everest-workspace* directory the following command
 edm init
 ```
 
+Advanced users would eventually want to use their customized lib versions or
+branches of repos. This can be achieved by editing the workspace-config.yaml
+file in the root of the workspace. An **edm init** will setup your current
+workspace accordingly.
+
 Be advised that even if you remove a repository from the config file it WILL NOT be deleted from the workspace.
```

Comment:
Where does this belong to? Is this related to the everest-complete.yaml or the workspace-config.yaml?

Is this obsolete or does this still apply?

Reply:
I think this part is obsolete, what you can do as an "advanced user" is to just check out the branch(es) that you want of your libraries in the same workspace folder and edm will automatically use them during building

Reply:
Thanks, what I did now: As I understand you correcly, (advanced) users could still use that edm feature of handling branches in their workspaces. So, I just transformed this section in a "note" box, as an additional information.

Should we better remove it completely including all additional info about branch config in this paragraph or can you already approve?

Reply:
Hi @hikinggrass , as chatted yesterday, I deleted the part about workspace config to make it a hidden (non-supported) feature. Instead, I added the "clean" way of pulling a dedicated release version number.

Please review and eventually approve, if everything seems to look good.